src/components/offer-banner/OfferBanner.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import { Typography } from '@mui/material'
+import IconButton from '@mui/material/IconButton'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+import {
+  ButtonActions,
+  Offer,
+  ProficiencyLevelEnum,
+  VariantEnum
+} from '~/types'
+import { styles } from '~/components/offer-banner/OfferBanner.styles'
+import AppChip from '~/components/app-chip/AppChip'
+import AppButton from '~/components/app-button/AppButton'
+
+import { useTranslation } from 'react-i18next'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+import useBreakpoints from '~/hooks/use-breakpoints'
+
+interface OfferBannerProps {
+  offer: Offer
+  buttonActions: ButtonActions[]
+}
+
+const OfferBanner: FC<OfferBannerProps> = ({ offer, buttonActions }) => {
+  const { t } = useTranslation()
+  const { isDesktop } = useBreakpoints()
+  const { author, authorFirstName, authorLastName, subject, proficiencyLevel } =
+    offer
+
+  const buttons = buttonActions.map((elem, index) => (
+    <AppButton
+      fullWidth
+      key={elem.label}
+      onClick={elem.handleClick}
+      sx={styles.button}
+      variant={index !== 0 ? VariantEnum.Tonal : VariantEnum.Contained}
```

Comment:
we dont use this approach anymore. variant should be specified in array, btw contained is by default , no need to pass it